internal/benchrunner/runners/common/scenario.go

```diff
@@ -0,0 +1,119 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package common
```

Comment:
Nit. `common` seems too generic, could this go directly in the `internal/benchrunner/runners` package? Or to an `internal/benchrunner/scenario` package?

Reply:
@aspacca Any thoughts?